### ISSUE TITLE
Training log

### ISSUE THREAD

**Issue by JK2JK** [cedricgsh/ISDNet#4]
Thanks for the source code! I tried to reproduce results in the paper, but I only achieved 72.3% on DeepGlobe dataset. Could you please provide your training log?

**Comment by cedricgsh** [comment]
Sorry，we didn' t save the training log. And, 72.3% is the test set results? The result shown in training log is validation set result, not the test set results.

**Comment by JK2JK** [comment]
> 

Thanks for your quick reply!   72.3% is the test set results. 

Our training script:  sh tools/dist_train.sh configs/isdnet/isdnet_r18-d8_1224x1224_80k_DeepGlobe.py 4

And test script:  python tools/test.py configs/isdnet/isdnet_r18-d8_1224x1224_80k_DeepGlobe.py  work_dirs/isdnet_r18-d8_1224x1224_80k_DeepGlobe/latest.pth --eval mIoU

Our results on test set:
![image](https://user-images.githubusercontent.com/115603322/195243400-221c1f1e-2800-4203-bb7e-5b264623830e.png)
![image](https://user-images.githubusercontent.com/115603322/195243399-930343df-6bd0-4b75-bc41-547fc1943c72.png)

**Comment by JK2JK** [comment]
> Sorry，we didn' t save the training log. And, 72.3% is the test set results? The result shown in training log is validation set result, not the test set results. Can you show your results?

Thanks for your quick reply! 72.3% is the test set results.

Our training script: sh tools/dist_train.sh configs/isdnet/isdnet_r18-d8_1224x1224_80k_DeepGlobe.py 4

And test script: python tools/test.py configs/isdnet/isdnet_r18-d8_1224x1224_80k_DeepGlobe.py work_dirs/isdnet_r18-d8_1224x1224_80k_DeepGlobe/latest.pth --eval mIoU

Our results on test set:
![image](https://user-images.githubusercontent.com/115603322/195243497-ec2f2dd2-9eed-4533-8f9f-cb64f02ac56a.png)
![image](https://user-images.githubusercontent.com/115603322/195243512-91c6f0b7-50d1-4f74-b50f-41a3ad4e7e06.png)

**Comment by JK2JK** [comment]
> Sorry，we didn' t save the training log. And, 72.3% is the test set results? The result shown in training log is validation set result, not the test set results. Can you show your results?

Thanks for your quick reply! 72.3% is the test set results.

Our training script: sh tools/dist_train.sh configs/isdnet/isdnet_r18-d8_1224x1224_80k_DeepGlobe.py 4

And test script: python tools/test.py configs/isdnet/isdnet_r18-d8_1224x1224_80k_DeepGlobe.py work_dirs/isdnet_r18-d8_1224x1224_80k_DeepGlobe/latest.pth --eval mIoU

Our results on test set:
![image](https://user-images.githubusercontent.com/115603322/195244194-006d4af4-83fb-4c77-8f69-5caf10e251ba.png)
![image](https://user-images.githubusercontent.com/115603322/195244208-fa91507a-0a62-412d-b08c-9887f7a8dd89.png)

**Comment by cedricgsh** [comment]
I cannot shared my pretained models due to patent. And you can try inrial Aerial dataset. If this dataset also fail to reproduce, I will ask my mentor to share pretained models.

**Comment by cedricgsh** [comment]
Try this pretrained [model.](https://drive.google.com/file/d/1wMKW7B_r6KA2bBYr9ki2R8WmLLusFMm4/view?usp=sharing)

**Comment by cedricgsh** [comment]
@cnstt Pretrained models are available now. You can find the download url in README file.

**Comment by cnstt** [comment]
> @cnstt Pretrained models are available now. You can find the download url in README file.

Thanks a lot for your fast reply and for making the pretrained model available. This is helping us a lot!
We encountered some issue during training using the standard dataset and model, but I will open another issue,  issue #9, to discuss about this :)

**Comment by st9song** [comment]
> 

你好，可以分享一下镜像吗

**Comment by songcv** [comment]
> 
 你好，可以分享一下镜像吗   @JK2JK
